packages/ember-htmlbars/lib/node-managers/view-node-manager.js

```diff
@@ -122,6 +126,10 @@ ViewNodeManager.prototype.rerender = function(env, attrs, visitor) {
       env.renderer.willUpdate(component, snapshot);
 
       if (component._renderNode.shouldReceiveAttrs) {
+        if (component.propagateAttrsToThis) {
```

Comment:
When is `propogageAttrsToThis` not present?  If this is intended to exclude `isGlimmerComponent` then we should check that instead.

As this is implemented, it might be possible to simply `Ember.Component.extend({ propogateAttrsToThis: null });` to opt out of the attrs propagation work which I believe is not desired.